demo.py

```diff
@@ -94,6 +92,7 @@
                 species = [so4, pom, soa, bc, dst, ncl],
                 number = stats.loguniform(3e7, 2e12),
                 geom_mean_diam = stats.loguniform(0.5e-7, 1.1e-7),
+                log10_geom_std_dev = 1.6,
```

Comment:
This isn't quite right. geom_std_dev=1.6, so log10_geom_st_dev = np.log10(1.6)

Reply:
Oh, right. I can fix all of those. Been running around a bit too much.

Reply:
I'm not sure if it's bad form for me to fix it myself or not! I was just looking for it. :)

Reply:
I think there's a way to suggest a fix, which can be committed right from here. We should sit down and work through a code review when it's convenient. (I have a meeting in a few minutes, so I just wanted to get these in!)